src/www/containers/settingspage/SettingsPage.jsx

```diff
@@ -0,0 +1,438 @@
+                    >
+                      <span>{t("settings.username")}</span>
+                      <input
+                        type="text"
+                        name="username"
+                        className="Username-field"
+                        onChange={(e) =>
+                          this.setState({ username: e.target.value.trim() })
+                        }
+                      />
+                      <span>{t("settings.favourite_categories")}</span>
+                      <Select
+                        className="Select"
+                        closeMenuOnSelect={false}
+                        components={animatedComponents}
+                        isMulti
+                        options={this.formatCategories(categoriesList, options)}
+                        defaultValue={this.formatCategories(
+                          favouriteCategories,
+                          favourite_options
+                        )}
+                        onChange={this.changeFavourites}
+                      />
+                      <span className="bio">{t("settings.bio")}</span>
+                      <textarea
+                        type="text"
+                        name="bio"
+                        className="Bio-field"
+                        defaultValue="Hello World"
+                        onChange={(e) =>
+                          this.setState({ bio: e.target.value.trim() })
+                        }
+                      />
+                      <span>{t("settings.location")}</span>
+                      <Map
+                        className="Map"
+                        center={this.state.position}
+                        zoom={13}
+                        onClick={(e) => this.setState({ position: e.latlng })}
+                      />
+                      <input
+                        type="submit"
+                        className="Submit-button"
+                        value={t("settings.update.profile")}
+                      />
+                    </form>
+                  </div>
+                  <div className="photo">
+                    <img src="/img/fakedata/profilePhoto.png" alt="Avatar" />
+                    <form
+                      id="profile-picture"
+                      method="post"
+                      action="/settings/submit"
+                      onSubmit={this.submitPicture.bind(this)}
+                    >
+                      <ImageUploader
+                        withIcon={false}
+                        buttonText={t("settings.buttonText")}
+                        imgExtension={[".jpg", ".png"]}
+                        label={t("settings.label")}
+                        maxFileSize={5242880}
+                        singleImage={true}
+                        withPreview={true}
+                        onChange={(pic) => {
+                          this.setState({
+                            profilePicture: pic,
```

Comment:
Does this set the profilePicture field to a string? Cause this is how you're handling it later on during submit.